data/config.json

```diff
@@ -1,42 +1,84 @@
-        "2": "data/ALL.chr2.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "3": "data/ALL.chr3.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "4": "data/ALL.chr4.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "5": "data/ALL.chr5.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "6": "data/ALL.chr6.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "7": "data/ALL.chr7.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "8": "data/ALL.chr8.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "9": "data/ALL.chr9.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "10": "data/ALL.chr10.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "11": "data/ALL.chr11.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "12": "data/ALL.chr12.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "13": "data/ALL.chr13.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "14": "data/ALL.chr14.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "15": "data/ALL.chr15.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "16": "data/ALL.chr16.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "17": "data/ALL.chr17.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "18": "data/ALL.chr18.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "19": "data/ALL.chr19.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "20": "data/ALL.chr20.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "21": "data/ALL.chr21.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz",
-        "22": "data/ALL.chr22.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz"
+        "1": "data/train/chr1.vcf.gz",
+        "2": "data/train/chr2.vcf.gz",
+        "3": "data/train/chr3.vcf.gz",
+        "4": "data/train/chr4.vcf.gz",
+        "5": "data/train/chr5.vcf.gz",
+        "6": "data/train/chr6.vcf.gz",
+        "7": "data/train/chr7.vcf.gz",
+        "8": "data/train/chr8.vcf.gz",
+        "9": "data/train/chr9.vcf.gz",
+        "10": "data/train/chr10.vcf.gz",
+        "11": "data/train/chr11.vcf.gz",
+        "12": "data/train/chr12.vcf.gz",
+        "13": "data/train/chr13.vcf.gz",
+        "14": "data/train/chr14.vcf.gz",
+        "15": "data/train/chr15.vcf.gz",
+        "16": "data/train/chr16.vcf.gz",
+        "17": "data/train/chr17.vcf.gz",
+        "18": "data/train/chr18.vcf.gz",
+        "19": "data/train/chr19.vcf.gz",
+        "20": "data/train/chr20.vcf.gz",
+        "21": "data/train/chr21.vcf.gz",
+        "22": "data/train/chr21.vcf.gz"
```

Comment:
Should this be chr22.vcf.gz instead of chr21.vcf.gz?

Reply:
I got rid of 21, 22, and 7 - we don't use those chromosomes.